wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -49,6 +49,14 @@
 _HashItems = Sequence[Optional[ast.AST]]
 
 
+#: names of functions in which we allow strings with modulo patterns.
+_modulo_pattern_exceptions: ClassVar[FrozenSet[str]] = frozenset((
```

Comment:
This is a module-level, isn't it? 🤔 

nitpick: `#: n` -> `#: N`

Reply:
Ah god dammit I'm dumb